### DISCUSSION TITLE
2x speed after long press doesn't work anymore

### DISCUSSION THREAD

**Discussion by depuntism** [ReVanced/revanced-discussions#1099] [Question] [answered]
Hello all,

Am I the only one facing the issue that the 2x speed after long press don't work after the latest patch?

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
https://github.com/ReVanced/revanced-patches/issues/54

Change the speed to 1.0x and the tap to hold will work.

Or you can repatch and exclude the playback speed patch and then tap to hold should work with any speed.

**Reply by depuntism** [reply]
I cannot seem to find a setting for that...

**Comment by 0-BlackSpectrum-0** [comment]
> https://github.com/ReVanced/revanced-patches/issues/54
> 
> Change the speed to 1.0x and the tap to hold will work.
> 
> Or you can repatch and exclude the playback speed patch and then tap to hold should work with any speed.

Hello, for me even though press to swipe gesture is disabled,
![SmartSelect_20250510-004210.jpg](https://github.com/user-attachments/assets/c1794804-d35f-46cf-9fae-4889312bc60a)

Long press to 2x still doesn't work, instead it shows this(also the video is playing at 1x)
![Screenshot_20250510-004228.png](https://github.com/user-attachments/assets/2c931402-f1dd-4643-896a-07516bea3c97)


Is this a bug or long press to 2x was removed?

**Reply by LisoUseInAIKyrios** [reply]
Turn off `Seekbar > Enable slide to seek`

**Reply by 0-BlackSpectrum-0** [reply]
> Turn off `Seekbar > Enable slide to seek`

Thanks it works now, but in previous patches, yt was able to do swipe to seek(not the long press swipe) and long press to 2x speed both, why does it not support now 🥲

**Reply by LisoUseInAIKyrios** [reply]
It was never intended to do both.

Patch an older version of YT and maybe it will still act that way.
